USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -118,14 +189,65 @@ private string RenderMarker(Marker input)
                     output.AppendLine(mTMarker.Title);
                     output.AppendLine("</div>");
                     break;
+                case FMarker fMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker);
+                    }
+                    break;
+                case FTMarker fTMarker:
+                    string footnote="";

```

Comment:
When concatenating strings in a loop like this always use a StringBuilder never concatenate the string directly. You'll end up with greater performance and less garbage on the stack that needs to be collected later.

Here are the docs on string builder if you want to read up on it.
https://docs.microsoft.com/en-us/dotnet/api/system.text.stringbuilder?view=netframework-4.8